src/router.js

```diff
@@ -227,21 +224,13 @@ const router = new Router({
   },
 });
 
-store.watch(
-  () => store.getters["auth/loggedIn"],
-  () => {
-    if (!store.getters["auth/loggedIn"]) {
-      router.push({ name: "login" });
-    }
-  },
-);
```

Comment:
This moved to the auth pinia store. We can't use the pinia stores before vue has been initialized, so keeping this here wasn't an option.